docs/features/software-catalog/well-known-annotations.md

```diff
@@ -2,9 +2,9 @@
 id: well-known-annotations
 title: Well-known Annotations on Catalog Entities
 sidebar_label: Well-known Annotations
-description: Documentation on lists a number of well known Annotations, that
+description: Documentation that lists a number of well known Annotations, that
 have defined semantics. They can be attached to catalog entities and consumed
-by plugins as needed
+by plugins as needed.
```

Comment:
This is a strange YAML syntax. I experimented a bit with docusaurus and I didn't find a solution for multiline strings in the frontmatter.

It also looks wrong in the page description:
![image](https://user-images.githubusercontent.com/720821/99243980-bdd71580-2801-11eb-9fc0-436de2cb1bf4.png)

Looks like this is the only thing that works:

```yaml
description: Documentation that lists a number of well known Annotations, that have defined semantics. They can be attached to catalog entities and consumed by plugins as needed.
```

Since this is also present in other files, it might be handled in a separate issue.

Reply:
Oh prettier be screwing things up for us I guess : /

Reply:
Looks like multiline strings in the frontmatter doesn't work in docusaurus <2 so this requires #3197 (or #2986).

Reply:
I fought with this for a while and failed - I actually didn't see the description update at all, it seemed to be taken from the first paragraph of text instead 🤷  Gonna merge for now and then hope that v2 will make it better